src/reducers/librarySlice.ts

```diff
@@ -0,0 +1,345 @@
+import * as toolkitRaw from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+import * as t from "../Types";
+import { localStorageOrDefault } from "../utils";
+
+// @ts-ignore
+const { createSlice } = toolkitRaw.default ?? toolkitRaw;
```

Comment:
So this piece is a bug with redux dealing with how ESM modules work in node. This has been fixed in v2.0.0, however that is still in alpha.

This is the work around for the time being:
https://github.com/reduxjs/redux-toolkit/issues/1960